### PR TITLE
add ns3777k prometheus recipe

### PR DESCRIPTION
**Pull request by ns3777k**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: ns3777k/prometheus-bundle/1.0/config/routes/metrics.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+metrics:
```
Comment (Nyholm) [inline ns3777k/prometheus-bundle/1.0/config/routes/metrics.yaml]: Could you rename this file to `ns3777k_prometheus .yaml` or `ns3777k_prometheus_metrics.yaml`?

File: ns3777k/prometheus-bundle/1.0/config/packages/ns3777k_prometheus.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ns3777k_prometheus:
+    namespace: app
+    adapter: apcu
+
+    listener:
+        enabled: true
```
Comment (Nyholm) [inline ns3777k/prometheus-bundle/1.0/config/packages/ns3777k_prometheus.yaml]: This line is not needed, right? Or do you expect a large number of the users to disable this?

File: ns3777k/prometheus-bundle/1.0/config/routes/metrics.yaml (hunk 1 above)
Comment (ns3777k) [reply ns3777k/prometheus-bundle/1.0/config/routes/metrics.yaml]: yup

File: ns3777k/prometheus-bundle/1.0/config/packages/ns3777k_prometheus.yaml (hunk 2 above)
Comment (ns3777k) [reply ns3777k/prometheus-bundle/1.0/config/packages/ns3777k_prometheus.yaml]: not sure. i removed the line :smile:

**Pull request merged**